tests/Functional/Execution/AbstractPromiseTest.php

```diff
@@ -22,19 +22,18 @@ class AbstractPromiseTest extends TestCase
 
     /**
      * isTypeOf used to resolve runtime type for Interface
-     *
-     * @throws \Digia\GraphQL\Error\InvariantException
-     * @throws \Digia\GraphQL\Error\SyntaxErrorException
      */
     public function testIsTypeOfUsedToResolveFunctionForInterface()
     {
+        /** @noinspection PhpUnhandledExceptionInspection */
```

Comment:
I'm not a fan of these, why not just keep the throws tags?

Reply:
It's a test, they shouldn't throw anything.